src/index.scss

```diff
@@ -0,0 +1,683 @@
+                line-height: 80px;
+            }
+
+            .main__feedback-form-input-email {
+                margin-top: 40px;
+                padding: 0 20px;
+                min-width: fit-content;
+                height: 50px;
+                background: black;
+                opacity: 0.8;
+                transition: .5s opacity;
+                border: 2px solid $main_text_color;
+                line-height: 40px;
+
+                &::placeholder {
+                    color: $main_text_color;
+                }
+            }
+
+            .main__feedback-form-input-phone {
+                margin-top: 20px;
+                padding: 0 20px;
+                min-width: fit-content;
+                height: 50px;
+                background: black;
+                opacity: 0.8;
+                transition: .5s opacity;
+                border: 2px solid $main_text_color;
+                line-height: 40px;
+
+                &::placeholder {
+                    color: $main_text_color;
+                }
+            }
+
+            .main__feedback-form-input-message {
+                margin-top: 20px;
+                padding: 0 20px;
+                min-width: fit-content;
+                height: 100px;
+                background: black;
+                opacity: 0.8;
+                transition: .5s opacity;
+                border: 2px solid $main_text_color;
+                line-height: 40px;
+
+                &::placeholder {
+                    color: $main_text_color;
+                }
+            }
+
+            .main__feedback-form-send-button {
+                margin-top: 25px;
+                width: 220px;
+                height: 55px;
+                border: none;
+                color: black;
+                background-color: $main_text_color;
+                cursor: pointer;
+
+                &:hover {
+                    transition: .5s background-color;
+                    color: $main_text_color;
+                    background-color: black;
+                    border: 1px solid $main_text_color;
+                }
+            }
+        }
+    }
+}
+
+
+
+.footer {
+    display: flex;
+    justify-content: space-between;
+
+    .footer__data {
+        display: flex;
+        align-items: center;
+        margin: 50px 0 50px 20px;
+        height: 50px;
+
+        .footer__data-element {
+            margin-left: 20px;
+
+            &:first-child {
+                margin-left: 0;
+            }
+        }
+    }
+
+    .footer__text {
+        display: flex;
+        align-items: center;
+        margin: 50px 0;
+        height: 50px;
+    }
+
+    .footer__social {
+        display: flex;
+        align-items: center;
+        margin: 50px 20px 50px 0;
+        height: 50px;
+
+        .footer__social-link {
+            margin-left: 20px;
+
+            &:first-child {
+                margin-left: 0;
+            }
+        }
+    }
+}
+
+@media(max-width: 768px) {
```

Comment:
That's better to separate such values in constants somewhere in _variables.scss, not hardcode them in multiple places of code.